app/views/works/_work_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<section class = "form_partial">
+  <h2>
+  <%= action_title %>
+  </h2>
+
+  <%= form_with model: model do |f| %>
+
+    <%= f.label :category %>
+    <br>
+      <%= f.select :category, ["movie", "book", "album"] %>
```

Comment:
This works for our given information! Just a thought experiment: instead of hard-coding an array into this form, is there a way we can pull this array from a controller somewhere, in case the list of categories changes over time?